.changeset/rude-yaks-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-storage": patch
+---
+
+chore(gen2 storage): update StorageImage deprecation warnings to reference docs.
```

Comment:
Probably don't need this one?

Reply:
Let's clean it up the changsets after merging to `main`?